client/src/js/components/bhEmployeeFinder.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * The employee finder component
+ *
+ * @module components/bhEmployeeFinder
+ *
+ * @description a component to deal with employee, it let you choose a employee
+ * 
+ **/
+ 
+//An employee finder component controller
+ 
+function employeeFinderController (employeeService){ 	
+ 	var ctrl = this;
+
+ 	ctrl.searchOptions = [
+ 		{key : 'code', label : 'EMPLOYEE.CODE', placeHolder : 'EMPLOYEE.SEARCH_BY_CODE'},
+ 		{key : 'names', label : 'EMPLOYEE.NAME', placeHolder : 'EMPLOYEE.SEARCH_BY_NAME'}
+ 	];
+
+ 	function load (){
+ 		ctrl.selectedEmployee = null;
+ 		ctrl.employeeValue = null;
+ 		ctrl.session = {state : 'finding'};
```

Comment:
Why do you need to use `session`?  Wouldn't it better to just call this property `ctrl.state`?